src/EPR.Accreditation.Facade.Common/RESTservices/HttpSiteService.cs

```diff
@@ -27,5 +26,13 @@ public class HttpSiteService : BaseHttpService, IHttpSiteService
         {
             await Put($"{accreditationExternalId}/Site", site);
         }
+
+        public async Task<Guid> CreateSite(
```

Comment:
Why do you need to create a site? It should exist at point of adding the address